src/libratbag-util.h

```diff
@@ -259,7 +259,6 @@ dpi_list_from_string(const char *str)
 
 	if (!str || str[0] == '\0')
 		return NULL;
-	printf("%s: ", str);
```

Comment:
nitpick: I do not think this change should be part of this commit. Please extract it in a separate one so we get a justification on why it is required.